runtime/cel/status_reader_test.go

```diff
@@ -34,47 +34,6 @@ import (
 	"github.com/fluxcd/pkg/runtime/cel"
 )
 
-func TestNewStatusReader(t *testing.T) {
```

Comment:
This test was just testing the error in constructing the evaluator internally, now it is injected, so the test is no longer needed.